### ISSUE TITLE
Spacing/sizing  is too small

### ISSUE THREAD

**Issue by vanishakesswani** [HaikuArchives/HaikuThemeManager#10]
![t2](https://cloud.githubusercontent.com/assets/24365311/21669486/1a169858-d332-11e6-9555-1a2f7cfaeb14.png)
This does not mach Haiku's user interface guidelines, nor does it look good.

**Comment by mmuman** [comment]
Indeed, the current UI needs to be redone with proper layout support, as it has a fixed window size.
